shopping-cart/render-line-items.js

```diff
@@ -0,0 +1,36 @@
+// import cart from '../data/cart.js';
+// import books from '../data/cart.js';
+
+export function renderLineItems(lineItem, book) {
+    
+    const tr = document.createElement('tr');
+    table.appendChild('tr');
```

Comment:
table isn't defined anywhere - you want to return tr instead, and append it to the table after you call the function